roles/ansible/tower/manage-organizations/templates/organization.j2

```diff
@@ -0,0 +1,6 @@
+{
+    "name": '{{ organization.name }}',
+    "description": '{{ organization.description | default('') }}',
+    "max_hosts": '{{ organization.max_hosts | default(0) | int }}',
+    "custom_virtualenv": '{{ organization.custom_virtualenv | default(omit) }}'
```

Comment:
some quick testing seems to indicate that `default(omit)` in a .j2 template won't work, ie.: it will generated something like:
```
"custom_virtualenv": '__omit_place_holder__da552b2518dc68d31b9249b8862910384af4b36c'
```

Reply:
nvm - looks like the `uri` module (and other modules) has built-in functionality to handle the `__omit_place_holder...`